web/src/pages/index.tsx

```diff
@@ -51,50 +42,82 @@ function HomepageHeader() {
 
         <div className={styles.carousel}>
           <Carousel>
-            <div className={styles.leftAligned}>
-              <div className="container">
-                <p>Sign up for our Newsletter</p>
-                <p className="hero__subtitle">
-                  Stay in touch with what's new with VAST
-                </p>
-                <div>
-                  <Link
-                    className="button button--info button--md"
-                    to="/newsletter"
-                  >
-                    Subscribe
-                  </Link>
+            {[
+              <div className={styles.blogPost}>
+                <div className="container">
+                  <p>Sign up for our Newsletter</p>
+                  <p className="hero__subtitle">
+                    Stay in touch with what's new with VAST
+                  </p>
+                  <div>
+                    <Link
+                      className="button button--info button--md"
+                      to="/newsletter"
+                    >
+                      Subscribe
+                    </Link>
+                  </div>
                 </div>
-              </div>
-            </div>
-            <BlogPostSlide
-              title={latestNonReleaseBlogPost.title}
-              link={latestNonReleaseBlogPost.link}
-            />
-            <BlogPostSlide
-              title={latestReleaseBlogPost.title}
-              link={latestReleaseBlogPost.link}
-              isRelease={true}
-            />
+              </div>,
+            ].concat(
+              blogpostsInternalArchive?.blogPosts
+                ?.slice(0, 3)
+                ?.map((post, idx) => (
+                  <BlogPostSlide
+                    key={idx}
+                    title={post?.metadata?.title}
+                    link={post?.metadata?.permalink}
+                    isRelease={post?.metadata?.tags
+                      ?.map((tag) => tag?.label)
+                      ?.includes('release')}
+                    imageLink={post?.metadata?.frontMatter?.image}
+                    description={post?.metadata?.frontMatter?.description}
+                  />
+                ))
+            )}
```

Comment:
Have you looked at how Docusaurus generates the excerpt? Here is the detail: https://docusaurus.io/docs/next/blog#blog-list. I didn't dig deeper, but my hope would be that we can reuse the truncation and simply get that content.